src/components/AboutProton/AboutProton.js

```diff
@@ -0,0 +1,49 @@
+import React from "react"
+import "../../pages/index.scss"
+import IconFb from "../../pictures/icon-fb.svg"
```

Comment:
Is it possible to import icons used in the design system? See #shape-mastodon in https://design-system.protontech.ch/icons/

(maybe a subset of the sprite used)

Reply:
You might ask @mmso to know if this is possible.